src/helpers/services.js

```diff
@@ -0,0 +1,80 @@
+import chat from '../utils/chat.png'
+import advisory from '../utils/advisory.png'
+import hotel from '../utils/best-place.png'
+import vacation from '../utils/sunlight.png'
+import flight from '../utils/aircraft.png'
```

Comment:
I wish you can create assets folder to put all images so that we can use utils for utility functions such as axios configurations and others